Только в /root/dk_scripts/tmp/1: 2-Feb-2016065513LINK-W-Downfa14KTCS0-SW7-24_192.168.22.87.conf
Только в /root/dk_scripts/tmp/1: 4-Jun-2013050521LINK-I-Upfa3KLBR31-SW3-24_192.168.22.202.conf
Только в /root/dk_scripts/tmp/2: LBR31-SW3-24_192.168.22.202.conf
diff -r /root/dk_scripts/tmp/2/TCS0-SW1-24_192.168.22.62.conf /root/dk_scripts/tmp/1/TCS0-SW1-24_192.168.22.62.conf
98a99
>  switchport access vlan 66 
diff -r /root/dk_scripts/tmp/2/TCS0-SW2-08_192.168.22.76.conf /root/dk_scripts/tmp/1/TCS0-SW2-08_192.168.22.76.conf
57a58,60
> interface fastethernet7
> switchport trunk native vlan 66 
> exit
Только в /root/dk_scripts/tmp/2: TCS0-SW7-24_192.168.22.87.conf
Только в /root/dk_scripts/tmp/2: TCS4-SW1-48_192.168.22.71.conf
